cfme/tests/services/test_service_catalogs.py

```diff
@@ -236,3 +239,57 @@ def test_order_service_after_deleting_provider(appliance, provider, setup_provid
     assert view.details.request_details.get_text_of("Request State") == "Finished"
     last_msg = f"Error: Source Template/Vm with id [{template_id}] has no EMS, unable to provision"
     assert view.details.request_details.get_text_of("Last Message") == last_msg
+
+
+@pytest.mark.tier(2)
+@pytest.mark.provider([VMwareProvider], scope="module")
```

Comment:
```suggestion
@pytest.mark.provider([InfraProvider], scope="module", selector=ONE)
```
This will create test like - `test_xyz[virtualcenter-datastore]`

The test currently has a name like `test_xyz[virtualcenter-6.5-datastore]`. If the name of this provider changes, the test ID and name will also change which we want to avoid as fas as possible.

Reply:
@dgaikwad I agree with @valaparthvi here, we should change this to `InfraProvider`